FabLabKasse/faucardPayment/FAUcardPaymentThread.py

```diff
@@ -319,9 +324,9 @@ def check_last_transaction(cur, con):
             return False
 
         if value[0] is magpos.codes.OK:  # Last transaction was successful
-            logging.warning("CheckTransaction: Kassenterminal vor erfolgreicher Buchung abgestürzt.")
+            logging.error("CheckTransaction: Kassenterminal vor erfolgreicher Buchung abgestürzt.")
             MagPosLog.save_transaction_result(cur, con, value[1], Decimal(value[2])/100, Info.transaction_ok.value)
-            logging.warning(u"CheckTransaction: Buchung für Karte {0} über Betrag {1} EURCENT fehlt".format(value[1], value[2]) )
+            logging.error(u"CheckTransaction: Buchung für Karte {0} über Betrag {1} EURCENT fehlt".format(value[1], value[2]) )
```

Comment:
Thats correct. The return value should only signal if the check could be performed without issues and does not correlate with the fact if a error was found. I updated the documentation and substituted the `return True` with `pass`